pyaedt/modules/MaterialLib.py

```diff
@@ -463,15 +463,26 @@ def add_material_sweep(self, swargs, matname):
         return index
 
     @pyaedt_function_handler()
-    def duplicate_material(self, material, new_name):
+    def duplicate_material(self, material_name, new_name=None, props=None):
         """Duplicate a material.
 
         Parameters
         ----------
-        material : str
+        material_name : str
             Name of the material.
         new_name : str
-            Name for the copy of the material.
+            Name for the copy of the material. If a new name is not specified,
+            the new material name is ``material_name + "_clone"``.
+        props : list
+            List of properties to parameterize when the material is duplicated.
+            Parameterized properties have project scope. Options are:
+
+            - 'permittivity'
+            - 'permeability'
+            - 'conductivity'
+            - 'dielectric_loss_tan'
+            - 'magnetic_loss_tan'
```

Comment:
```suggestion
            - `'permittivity'`
            - `'permeability'`
            - `'conductivity'`
            - '`dielectric_loss_tan'`
            - '`magnetic_loss_tan'`
```
Double back ticks are required. Otherwise, instead of being rendered as code entities in the doc, they are simply italicized.